routes/accounts/register.js

```diff
@@ -0,0 +1,9 @@
+const encryptPassword = require( '../../auth/encryptPassword' )
+const RegistrationEmail = require( '../../src/mail/registration_email' )
+
+const register = ( User, email, password ) => {
```

Comment:
Since this is a single expression that we return, please omit the braces and `return` keyword